composer.json

```diff
@@ -18,6 +18,7 @@
     "require": {
         "php": ">=7.2.5",
         "ext-xml": "*",
+        "dragonmantank/cron-expression": "^2.3",
```

Comment:
On the root composer.json, this dependency should probably be moved to `require-dev`. Also, there's a version 3 of that library available. Can we upgrade?

Reply:
Moved to `3.0`